### DISCUSSION TITLE
Markdown Blockquote Handling in a less annoying way.

### DISCUSSION THREAD

**Discussion by LambdAurora** [stoatchat/discussions#81] [Feature Suggestions]
## So, what's the actual issue?

Revolt Chat uses the `markdown-it` library for markdown rendering, but it has a flow in its default blockquote handling.
But first, what is a markdown Blockquote?

A markdown block quote can be made by using the character `>` then typing the stuff to be quoted next.
Example:
```markdown
> Quotation! Yay

>no space before

> Can be multiline!
> ooo
```
Rendered:
> Quotation! Yay

>no space before

> Can be multiline!
> ooo

As you may see, if no space is included, it will still consider it a quote, which becomes a bit annoying in a discussion app and is a bit different from Discord's "markdown" handling.
Discord will require a space/tab for it to consider a quote, which is great for some emoticons like `>w<`.
The issue is with the current handling `>w<` will be considered a quote of `w<`, and requiring users to type a backslash before is a bit annoying and reduce the speed of typing.

## Ok, but how do we solve it?

To solve this, Revolt would need to require that space/tab, this can either be achieved by overriding entirely `markdown-it`'s block quote handling, or maybe it includes an option to require a space/tab but I'm sadly not aware of this currently.

I hope this can be considered in the future!

**Comment by Yu-Vitaqua-fer-Chronos** [comment]
This would be great because we use emoticons like that frequently aha
